### PR TITLE
Feature/web about font

### PR DESCRIPTION
**Pull request by ryan-schuster**
### Description

Changed the font of the 'about' window pop-up from old-fashioned 2000's computer font to times new roman.

### What are you changing?

*A font

### Did you increase testing or code coverage?

No

### Did you add any dependencies to the project?

No

### Did you update the relevant documentation?

No, not big enough change.

### CODE REVIEW

**Review by pranayk2610 — CHANGES_REQUESTED** [review]
Merge into release

File: web/ugahacks-7/components/Window/Window.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      );
+    case "image":
+      return <p></p>; // Add similar structure to chat layout with mapping over an obj with image data - See Sponsors section on Hacks 6
+  }
+};
+
+const Window = ({
+  windowTitle,
+  width = "50vh",
+  height = "50vh",
```
Comment (genki-aik) [inline web/ugahacks-7/components/Window/Window.tsx]: Did we not change this file and set height to auto?

File: web/ugahacks-7/styles/Window.module.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  font-size: 1.25em;
+}
+.bodyText {
+  font-size: 2em;
+}
+
+.blueBanner, .url, .bodyText, .heading {
+  font-family: 'Times New Roman', Times, serif;
+  padding-left: 2px;
+}
```
Comment (genki-aik) [inline web/ugahacks-7/styles/Window.module.css]:
```suggestion
}

```

File: web/ugahacks-7/components/Window/Window.tsx (hunk 1 above)
Comment (genki-aik) [reply web/ugahacks-7/components/Window/Window.tsx]: I guess we will just solve the merge conflicts in master if we just push this?

**Review by genki-aik — COMMENTED** [review]
I left some comments

File: web/ugahacks-7/components/Window/Window.tsx (hunk 1 above)
Comment (pranayk2610) [reply web/ugahacks-7/components/Window/Window.tsx]: yeah i think thats a minor thing

**Review by genki-aik — APPROVED** [review]

**Review by genki-aik — APPROVED** [review]

**Review by genki-aik — APPROVED** [review]

**Review by pranayk2610 — APPROVED** [review]

**Pull request merged**